diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index b56b979ee8ca5f21abb8245b2b1f38ddd7393ac1..0000000000000000000000000000000000000000
diff --git a/dist/manifest/validation.cjs b/dist/manifest/validation.cjs
index 340b872b28944ad62d983769e3a7f589bf8d6867..ba1581d2ed7a0a08c2593aa509afb8a7c354ca7d 100644
--- a/dist/manifest/validation.cjs
+++ b/dist/manifest/validation.cjs
@@ -137,6 +137,7 @@ exports.SnapManifestStruct = (0, superstruct_1.object)({
     repository: (0, superstruct_1.optional)((0, superstruct_1.object)({
         type: (0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity),
         url: (0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity),
+        directory: (0, superstruct_1.optional)((0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity)),
     })),
     source: (0, superstruct_1.object)({
         shasum: utils_1.ChecksumStruct,
diff --git a/dist/manifest/validation.mjs b/dist/manifest/validation.mjs
index 30c9001894981644ed6f0806e453aa18fb38fbf6..a92688d07f562d98452b476c0a33377733cd2329 100644
--- a/dist/manifest/validation.mjs
+++ b/dist/manifest/validation.mjs
@@ -133,6 +133,7 @@ export const SnapManifestStruct = object({
     repository: optional(object({
         type: size(string(), 1, Infinity),
         url: size(string(), 1, Infinity),
+        directory: optional(size(string(), 1, Infinity)),
     })),
     source: object({
         shasum: ChecksumStruct,
diff --git a/dist/types.cjs b/dist/types.cjs
index 09f39e6ea1f8c951bce3333c55140ee4b977370a..3c8588c880c3bdc3aedfd8f92b0f997fbd0246de 100644
--- a/dist/types.cjs
+++ b/dist/types.cjs
@@ -18,6 +18,7 @@ exports.NpmSnapPackageJsonStruct = (0, superstruct_1.type)({
     repository: (0, superstruct_1.optional)((0, superstruct_1.object)({
         type: (0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity),
         url: (0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity),
+        directory: (0, superstruct_1.optional)((0, superstruct_1.size)((0, superstruct_1.string)(), 1, Infinity)),
     })),
 });
 /**
diff --git a/dist/types.mjs b/dist/types.mjs
index 3115ccbc494db90349521a1b3e8699e819bff1bb..838e8293328ec40677505a6cd8aa67f980a81d8c 100644
--- a/dist/types.mjs
+++ b/dist/types.mjs
@@ -15,6 +15,7 @@ export const NpmSnapPackageJsonStruct = type({
     repository: optional(object({
         type: size(string(), 1, Infinity),
         url: size(string(), 1, Infinity),
+        directory: optional(size(string(), 1, Infinity)),
     })),
 });
 /**
